src/declarations/stencil-public-compiler.ts

```diff
@@ -1845,6 +1845,27 @@ export interface OutputTargetDist extends OutputTargetBase {
   dir?: string;
 
   collectionDir?: string | null;
+  /**
+   * When `true` this flag will transpile aliased import paths defined in
```

Comment:
Just in case we do 'transpile' -> 'transform':
```suggestion
   * When `true` this flag will transform aliased import paths defined in
```